ocrd_eval.schema.yml

```
@@ -0,0 +1,192 @@
+      '@id':
+        type: string
+        format: uri
+        description: URL of the thing
+      label:
+        type: string
+        description: Description of the thing for UI purposes
+
+  EvaluationMetadata:
+    type: object
+    title: Metadata about one evaluation
+    additionalProperties: false
+    description: >
+      EvaluationMetadata contains all the info on how an EvaluationReport came to be.
+
+      There are two OCR-D *workflows* involved:
+        - ocr_workflow: The workflow which produced the OCR results to evaluate
+        - eval_workflow: The workflow run to evaluate OCR and GT
+
+      There are three OCR-D *workspaces* involved:
+        - gt_workspace: The workspace containing the GT
+        - ocr_workspace: The workspace containing the OCR results from ocr_workflow
+        - eval_workspace: The workspace on which the eval_workflow was run
+
+    required:
+      - ocr_workflow
+      - ocr_workspace
+      - eval_workflow
+      - eval_workspace
+      - gt_workspace
+      - document_metadata
+
+    properties:
+
+      ocr_workflow:
+        allOf:  [{ '$ref': '#/$defs/LabeledUrl' }]
+        description: The OCR-D workflow that produced the ocr_workspace
+
+      ocr_workspace:
+        allOf:  [{ '$ref': '#/$defs/LabeledUrl' }]
+        description: The workspace containing the OCR
+
+      eval_workflow:
+        allOf:  [{ '$ref': '#/$defs/LabeledUrl' }]
+        description: The OCR-D workflow that produced the eval_workspace
+
+      eval_workspace:
+        allOf:  [{ '$ref': '#/$defs/LabeledUrl' }]
+        description: The workspace containing the evaluation results
+
+      gt_workspace:
+        allOf:  [{ '$ref': '#/$defs/LabeledUrl' }]
+        description: The workspace containing the GT
+
+      workflow_steps:
+        type: object
+        description: Human readable description of the individual steps in the workflow (for UI)
+        patternProperties:
+          '^[0-9]+$':
+            type: string
+            description: Description of this workflow step
+
+      workflow_model:
+        type: string
+        description: Human readable name of the main model used for recognition in the OCR workflow (for UI)
+
+      eval_tool:
+        type: string
+        description: Human readable name and version of evaluation tool used (for UI
+
+      document_metadata:
+        type: object
+        title: Bibliographical and typographical metadata about the work to be evaluated
+        properties:
+
+          publication_year:
+            type: number
+            description: Year he document was originally published
+
+          publication_century:
+            type: string
+            description: Century he document was originally published
+            pattern: '[12][0-9]{3}-[12][0-9]{3}'
+
+          publication_decade:
+            type: string
+            description: Decade the document was originally published
+            pattern: '[12][0-9]{3}-[12][0-9]{3}'
```

```suggestion
            pattern: '[12][0-9]{2}0-[12][0-9]{2}0'
```
See comment(s) above.